includes/embeds/class-amp-vimeo-embed.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/embeds/class-amp-base-embed-handler.php' );
+
+// Much of this class is borrowed from Jetpack embeds
+class AMP_Vimeo_Embed_Handler extends AMP_Base_Embed_Handler {
+
+	const URL_PATTERN = '#https?:\/\/(www\.)?vimeo\.com\/*\/*\/.*#i';
```

Comment:
Since we're only handling URLs in the format `https://vimeo.com/172355597` should we limit this regex to something simpler like `'#https?:\/\/(www\.)?vimeo\.com\/\d+#i'`